pkg/controller/awsmachine/awsmachine_controller.go

```diff
@@ -153,116 +155,92 @@ func (r *ReconcileAWSMachine) Reconcile(request reconcile.Request) (reconcile.Re
 }
```

Comment:
As I was trying to figure out how the error messages are written out, since they are returned from `getOrCreate` and `reconcile`, it looks like we are using plain `klog` to write them out instead of using the logger from the scope above

Reply:
Switched those up to use the scope logger